src/Rings/AbelianClosure.jl

```diff
@@ -487,11 +487,90 @@ function Oscar.root(a::QabElem, n::Int)
   return mu
 end
 
+function Oscar.roots(f::PolyElem{QabElem})
+  Qab = base_ring(f)
+  c = reduce(lcm, map(conductor, coefficients(f)), init = Int(1))
```

Comment:
would `mapreduce` work here?
```suggestion
  c = mapreduce(conductor, lcm, coefficients(f), init = Int(1))
```